docs/contribute/benchmarks/TRAINING/benchmarks_DL2_to_classification.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>again, please add energy bias here. It's the same plot, but the mean of the distribution rather than the width. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/protopipe/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>DONE</p>